api/Performance.json

```diff
@@ -106,43 +496,223 @@
             },
             "firefox_android": {
-              "version_added": null
+              "version_added": "15",
+              "notes": "Starting with Firefox 57.0.4, the accuracy of this function has been reduced, first to <a href='https://blog.mozilla.org/security/2018/01/03/mitigations-landing-new-class-timing-attack/'>20 microseconds</a> and subsequently (in Firefox 59) to 2 milliseconds, while the Mozilla team investigate a security issue related to the precise timing of events. The precision may be improved again in a future release."
             },
             "ie": {
-              "version_added": null
+              "version_added": "10"
             },
             "opera": {
-              "version_added": null
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": "8"
+            },
+            "safari_ios": {
+              "version_added": "9"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "onresourcetimingbufferfull": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Performance/onresourcetimingbufferfull",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
             },
             "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "setResourceTimingBufferSize": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Performance/setResourceTimingBufferSize",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
```

Comment:
The [MDN article for this method](https://developer.mozilla.org/en-US/docs/Web/API/Performance/setResourceTimingBufferSize#Browser_compatibility) has more information, see above.